javascript/src/test/components/ChannelMessageReaction/MessageTableReaction.test.js

```diff
@@ -24,6 +38,369 @@ describe("MessageTableReactions tests", () => {
+        expect(testText).toBeInTheDocument();
+        expect(testCount).toBeInTheDocument();
+    });
+    test("row.message_reactions is formatted (channel mention)", () => {
+        useSWR.mockReturnValue({
+            data: [{
+                id: "U017218J9B3",
+                real_name: "test-person"
+            }]
+        });
+        const testMessages = [{
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "user_profile": {
+                "real_name": "Test Person"
+            },
+            "message_reactions": [{
+                count: 1,
+                name: "test-name"
+            }]
+        }];
+        const testReaction = "test-name";
+        const { getByText } = render(<MessageTableReaction messages = {testMessages} reaction = {testReaction}/>);
+        const testUser = getByText(/Test Person/);
+        const testText = getByText(/@test-person has joined the channel/);
+        const testCount = getByText(/1/);
+        expect(testUser).toBeInTheDocument();
+        expect(testText).toBeInTheDocument();
+        expect(testCount).toBeInTheDocument();
+    });
+    test("Username not found", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const testMessages = [{
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            },
+            "message_reactions": [{
+                count: 1,
+                name: "test-name"
+            }]
+        }];
+        const testReaction = "test-name";
+        const { getByText } = render(<MessageTableReaction messages = {testMessages} reaction = {testReaction}/>);
+        const nameElement = getByText(/@U017218J9B3/);
+        expect(nameElement).toBeInTheDocument();
+    });
+    test("Unembedded https links are clickable", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const testMessages = [{
+            "user": "U017218J9B3",
+            "text": "Meeting at <https://us05web.zoom.us/j/87975276031?pwd=QUkvLzBJblYwYzNtcnJ5anA0OHk1dz09>",
+            "user_profile": {
+                "real_name": "Test Person"
+            },
+            "message_reactions": [{
+                count: 1,
+                name: "test-name"
+            }]
+        }];
+        const testReaction = "test-name";
+        const { getByText } = render(<MessageTableReaction messages = {testMessages} reaction = {testReaction}/>);
+        const linkElement = getByText(/https:\/\/us05web.zoom.us\/j\/87975276031\?pwd=QUkvLzBJblYwYzNtcnJ5anA0OHk1dz09/);
+        expect(linkElement.href).toEqual("https://us05web.zoom.us/j/87975276031?pwd=QUkvLzBJblYwYzNtcnJ5anA0OHk1dz09");
```

Comment:
I don't know if this is a real Zoom URL (it appears to not work for me), but maybe we should use something like `example.zoom.us` instead as a placeholder?

Reply:
I changed it to the link to the Github repository link. It should work now.